docs/docs/recipes.md

```diff
@@ -94,6 +94,83 @@ There are so many ways to add styles to your website; Gatsby supports almost eve
 - Use the CSS-in-JS library [Styled Components](/docs/styled-components/)
 - Use [CSS Modules](/tutorial/part-two/#css-modules)
 
+### Using Styled Components
+
+#### Requirements
```

Comment:
```suggestion
#### Prerequisites
```

This PR: #15236 changed the name from Requirements to Prerequisites so probably worth updating.